giskard/scanner/llm/llm_chars_injection_detector.py

```diff
@@ -101,7 +101,6 @@ def _generate_char_injection_tests(issue: Issue):
     feature = issue.features[0]
     return {
         f"Character injection ({issue.meta['special_char'].encode('unicode_escape').decode('ascii')}) in “{feature}”": test_llm_char_injection(
-            model=issue.model,
```

Comment:
we don't want to fix the model? Is that also what we do in the traditional scan when we generate tests?

Reply:
After #1486 the default way is to let the scan set them at the test suite level. So that you can benchmark a different model on the same test suite (in particular from the Hub):

```py
report = gsk.scan(model)

suite = report.generate_test_suite()

suite.run(my_other_model)
```